### ISSUE TITLE
No verification email

### ISSUE THREAD

**Issue by AaronBlare** [automoss/automoss#4]
Hello!

I am trying to use AutoMOSS with Docker, however, I can't verify the email address - I do not receive a confirmation email. 

Could this problem be related to the fact that moss.stanford.edu is unavailable?

**Comment by iamricardo** [comment]
Hi Aaron, 

I am running into the same thing and was wondering if you had a work-around for it?

**Comment by MathieuIko** [comment]
Hi,

I also encountered this problem using the Docker version of AutoMOSS.

I have been able to get around this by manually validating the email address through the container's database, by following these steps:
- Get a prompt on the MySQL daemon of the container:
    - `docker exec -it name_of_your_container mysql`
- Ensure you are using the `automoss` database:
    - `use automoss;`
- Validate the email address by manually updating the field `is_verified`:
    - `UPDATE users_user SET is_verified=1 WHERE primary_email_address="target_address@example.com";`

**Comment by anudeep-17** [comment]
I am trying to reset my password, and i have never got an email, if you have any command for that?

**Comment by jonyejin** [comment]
I don't think you can reset password, but I did `delete from users_user where primary_email_address = "target_address@example.com"; and it worked
